.github/workflows/ci.yaml

```diff
@@ -10,32 +10,28 @@ on:
 jobs:
     cs-fix:
         name: Run code style check
-        runs-on: "ubuntu-20.04"
-        strategy:
-            matrix:
-                php:
-                    - '8.0'
+        runs-on: "ubuntu-22.04"
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
 
             - name: Setup PHP Action
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: ${{ matrix.php }}
+                  php-version: '8.0'
```

Comment:
```suggestion
                  php-version: '8.1'
```

Reply:
I used `8.0` because I wasn't aware that php-cs-fixer had support for 8.1. Thanks.